docs/main.css

```diff
@@ -0,0 +1,249 @@
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 20px 0px 10px;
+    background-color: rgba(174,174,174,0.65);
+}
+
+#npmpackage, #npmversion {
+    border-radius: 5px;
+    padding: 0px 10px 0px 0px;
+}
+.gh-btns {
+    border-radius: 5px;
+    padding: 0px 0px 0px 5px;
+}
+
+#npmpackage {
+    width: 160px;
+    height: 30px;
+}
+
+#npmversion {
+    width: 125px;
+    height: 30px;
+}
+
+/* Main Content flexbox */
+.main {
+    max-width: 640px;
+    margin: 0 auto;
+    padding: 50px 25px;
+}
+
+/* Info */
+.infoContainer {
+    margin: 0 auto;
+    margin-top: 55px;
+    width: 35%;
+}
+
+#infoTitle {
+    font-family: "Arial Black";
+    font-size: 24px;
+    color: #EE7C5B;
+}
+
+.contents {
+    font-family: "Arial";
+    font-size: 20px;
+    text-align: left;
+    line-height: 23px;
+}
+
+.contents:nth-of-type(1) {
+    margin-top: 30px;
+}
+
+.contents:nth-of-type(2) {
+    margin-top: 15px;
+}
+
+/* Require */
+#requireTitle {
+    margin-top: 50px;
+    font-family: "Arial Black";
+    font-size: 24px;
+    color: #EE7C5B;
+}
+
+#nodelogo {
+    padding: 10px 0px 0px 35px;
+    width: 230px;
+    height: 135px;
+}
+
+#gitlogo {
+    padding: 10px 0px 0px 35px;
+    width:300px;
+    height: 125px;
+}
+
+.requireLinks {
+    font-family: "Arial";
+    font-size: 24px;
+}
+
+.main p:nth-of-type(3) {
+    padding: 15px 0px 0px 0px;
+}
+
+a {
+    text-decoration: none;
+}
+
+/* Installation */
+#installTitle, #linuxMacTitle, #usageTitle {
+    margin-top: 50px;
+    font-family: "Arial Black";
+    font-size: 24px;
+    color: #EE7C5B;
+}
+
+#sourceTitle {
+    margin-top: 5px;
+    font-family: "Arial Black";
+    font-size: 20px;
+    color: #EE7C5B;
+}
+
+.contents:nth-of-type(5) {
+    margin-top: 20px;
+}
+
+pre {
+    padding: 10px;
+    background-color: #F6F8FA;
+    border-radius: 3px;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.contents:nth-of-type(6) {
+    margin-top: 10px;
+}
+
+.contents:nth-of-type(7) {
+    margin-top: 10px;
+}
+
+/* Linux/Mac */
+.contents:nth-of-type(9) {
+    margin-top: 20px;
+}
+
+/* Usage */
+.contents:nth-of-type(10) {
+    margin-top: 20px;
+}
+
+table {
+    margin-top: 10px;
+    width: 100%;
+    border: 2px solid #979797;
+    border-collapse: collapse;
+}
+
+td {
+    border: 2px solid #979797;
+    font-size: 18px;
+    padding: 5px 0px 5px 5px;
+}
+
+th {
+    border: 2px solid #979797;
+    font-size: 18px;
+    text-align: left;
+    color: #EE7C58;
+    padding: 5px 0px 5px 5px;
+}
+
+.description {
+    font-family: "Arial Black";
+}
+
+.command {
+    font-family: 'Courier New', Courier, monospace;
+}
+
+tr:nth-child(even) {
+    background-color: #D5D5D5;
+}
+
+/* Footer */
+footer {
+    height: 158px;
+    text-align: center;
+    color: #ed9e6d;
+    background: radial-gradient(100% 50%, #E0C851 0%, #E93F33 82%);
+}
+
+#contributing {
+    margin-top: 25px;
+    color: #ED9E6D;
+    padding: 10px 0px 10px 0px; 

```

Comment:
Shorthand values would be helpful here: https://developer.mozilla.org/en-US/docs/Web/CSS/padding